### PR TITLE
fix fmt error

### PR DESCRIPTION
**Pull request by windtalker**
### What problem does this PR solve?

Issue Number: ref #9087

Problem Summary:

### What is changed and how it works?

```commit-message

```

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by yibin87 — APPROVED** [review]
LGTM

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

File: dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp [hunk 1]
```diff
@@ -168,7 +168,7 @@ std::tuple<std::optional<RegionRetryList>, RegionException::RegionReadStatus> Ma
                         throw TiFlashException(
                             Errors::Coprocessor::BadRequest,
                             "Income key ranges is illegal, region_id={} version={} conf_version={} request_range=[{}, "
-                            "{}) region_range=[{}, {}}",
+                            "{}) region_range=[{}, {}]",
```
Comment (JaySon-Huang) [inline dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp:171]: We need to backport this to release-7.5 and release-8.1
https://github.com/pingcap/tiflash/blob/release-7.5/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L152-L157

**Pull request merged**